pages/deployments.md.erb

```diff
@@ -0,0 +1,146 @@
+
+The ":rocket:" step in the deployment example above uses the concurrency attributes `concurrency_group` and `concurrency_limit`. These work together to ensure that no matter how many builds are triggered on this pipeline at once, no ":rocket:" steps can run in parallel. It also prevents deploys clashing and attempting to write to the same resources. 
+
+<screenshot of a deploy pipeline>
+
+## Manual approval steps
+
+Adding a manual approval to your pipeline before your deployment ensures that a deploy never goes out without explicit approval.
+
+To create a manual approval for your deploy, add a block step before your deploy step or before the step that triggers your deploy pipeline. 
+
+The below example uses the same pipeline as the [Single deployment step](/docs/deployments#single-deployment-steps) section, but adds a block step before the step that performs the deploy:
+
+```
+steps:
+  - label: ":hammer:"
+    command: "scripts/test"
+  
+  - block: "Unblock deployment?"
+  
+  - label: ":rocket:"
+    command: "scripts/deploy"
+    branches: "master"
+    concurrency_group: "my-app-deploy"
+    concurrency_limit: 1
+```
+
+Until the block step is manually unblocked either in Buildkite or via an API call, the build will be paused and the ":rocket:" deployment step will not run.
+
+## External deployment systems
+
+As well as writing deployment scripts, you can use any external systems alongside Buildkite to perform your deployments.
+
+### Container services
+
+You can deploy images or containers to services like Kubernetes, Heroku, ECS, or Github Deployments. Deploy your containers directly from a script in a Buildkite command step using either the single deploy step pattern or the dedicated pipeline pattern discussed above.
+
+For a more detailed walk-through with examples, we have deployment guides for [Heroku](/docs/deployments/deploying-to-heroku) and [Kubernetes](/docs/deployments/deploying-to-kubernetes). 
+
+### Plugins
+
+There are [Buildkite plugins](/docs/pipelines/plugins) available for deployments. For example, the [ECS deploy plugin](https://github.com/buildkite-plugins/ecs-deploy-buildkite-plugin) and the [AWS lambda deploy plugin](https://github.com/envato/lambda-deploy-buildkite-plugin). 
+
+You can use these plugins in place of scripts to perform your deploys. They have parameters inbuilt to send all of the data required for a deployment.
+
+The below code is an example of a dedicated pipeline to deploy an image to ECS. It would be triggered by a pipeline that performed other actions like running tests and building the image.
+
+```
+steps:
+  - block: "Release to production?"
+  - label: ":ecs: :rocket:"
+    concurrency_group: "rails-app-deploy"
+    concurrency: 1
+    plugins:
+      - ecs-deploy#v1.3.0:
+        cluster: "production"
+        service: "app"
+        task-definition: "production-deploy/rails-app.json"
+        task-family: "rails-app"
+        image: "${ECR_REPOSITORY}/rails-app:${BUILDKITE_COMMIT}"
```

Comment:
We could simplify this a little:

```suggestion
        image: "an-ecr-repo/rails-app:${BUILDKITE_COMMIT}"
```

Aside from that, we could just use the same example from the ECS readme, but with the concurrency group?

Reply:
@plasticine and I started with the example code from the readme, but wanted to make it more realistic. I'm keen to keep it, unless it's got something wrong?